packages/block-library/src/pullquote/edit.js

```diff
@@ -70,6 +69,8 @@ function PullQuoteEdit( {
 					{ shouldShowCitation && (
 						<RichText
 							identifier="citation"
+							tagName={ isWebPlatform ? 'cite' : undefined }
+							style={ { display: 'block' } }
```

Comment:
Thanks @aristath, I am going to merge this as is since it matches the implementation of the Quote block exactly, so at least there is consistency. I will prepare a follow-up PR that goes through both blocks (and any others I find) and move these inline styles to the respective `style.scss` files. There is a decent amount of them.